### PR TITLE
[Chore] Design Improvements to Accordion Counter/Renamed Badge #901

### PR DESCRIPTION
**Pull request by Saravanan-Tangoe**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](https://github.com/mobi/goponents/blob/main/CONTRIBUTING.md)
- [x] Tests for the changes have been added
- [x] Docs have been added or updated

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Style Update (CSS)
- [ ] Other... Please describe:

What is the current behavior?
Issue Number: https://github.com/mobi/goponents/issues/901

What is the new behavior?
Introduces new badge pill component.
image

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.component.html [hunk 1]
```diff
@@ -1,139 +1 @@
-<header class="go-page-title">
```
Comment (swimtech20) [inline projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.component.html:1]: Just curious, where did all this code go? Was it not needed anymore? Seems like a lot of code to just be deleted...

File: projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.coponent.ts [hunk 2]
```diff
@@ -1,40 +1,17 @@
 import { Component } from '@angular/core';
+import { NavGroup } from '../../../../../../../go-lib/src/public_api';
```
Comment (swimtech20) [inline projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.coponent.ts:3]: The file name here is misspelled - should be `badge-docs.component.ts` (missing 'm')

File: projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/components/positioned-badge-docs/positioned-badge-docs.component.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
+<section class="go-container">
```
Comment (swimtech20) [inline projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/components/positioned-badge-docs/positioned-badge-docs.component.html:1]: oh - here is where the code went! Disregard prior comment lol

File: projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/components/positioned-badge-docs/positioned-badge-docs.component.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+})
+export class PositionedBadgeDocsComponent {
+  componentBindings: string = `
+  @Input() badgeData: string;
+  @Input() badgeColor: string = 'neutral';
+  @Input() displayData: boolean = true;
+  `;
+
+  iconBadge: string = `
+  <go-badge badgeData="12">
```
Comment (swimtech20) [inline projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/components/positioned-badge-docs/positioned-badge-docs.component.ts:16]: Please indent everything within the backticks for all these string objects below - not trying to nit-pick just trying to keep the code legible!

**Review by swimtech20 — CHANGES_REQUESTED** [review]
Looking good! Just a few comments, nothing major.

File: projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.component.html (hunk 1 above)
Comment (Saravanan-Tangoe) [reply projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.component.html:1]: its added to app-positioned-badge-docs

File: projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/components/positioned-badge-docs/positioned-badge-docs.component.ts (hunk 4 above)
Comment (Saravanan-Tangoe) [reply projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/components/positioned-badge-docs/positioned-badge-docs.component.ts:16]: we need above and below one liner space to keep string aligned inside the <code></code>.

Comment (swimtech20) [reply projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/components/positioned-badge-docs/positioned-badge-docs.component.ts]: oh okay, that makes sense

**Review by jaykravetz — APPROVED** [review]

**Review by swimtech20 — CHANGES_REQUESTED** [review]
Code is looking good! However, I was looking at the second screenshot in the Conversation tab - it looks like the go-pill and the code block are slightly misaligned... we should make the pill centered if not doing so already. Maybe the code block is the one misaligned, but I just want to make sure.

**Review by swimtech20 — APPROVED** [review]
Looks good!

**Pull request merged**